src/common/actions/snaps.js

```diff
@@ -39,3 +42,19 @@ export function removeSnap(repositoryUrl) {
     }
   };
 }
+
+export function fetchSnapStableRelease(repositoryUrl, snapName) {
+  const query = 'channel=stable&fields=name,revision';
```

Comment:
I'd prefer this to be built from an object with `qs.stringify` or similar; not because the current code is wrong, but in case somebody modifies or copies-and-pastes this in future and introduces unsafe parameter encoding.